app/swagger/requests/prescriptions.rb

```diff
@@ -0,0 +1,306 @@
+# frozen_string_literal: true
+module Swagger
+  module Requests
+    class Prescriptions
+      include Swagger::Blocks
+
+      swagger_path '/v0/prescriptions' do
+        operation :get do
+          key :description, 'Get a list of active and inactive prescriptions'
+          key :operationId, 'indexPrescriptions'
+          key :tags, %w(prescriptions)
+
+          parameter name: :page, in: :query, required: false, type: :integer,
+                    description: 'Page of results, greater than 0'
+
+          parameter name: :per_page, in: :query, required: false, type: :integer,
+                    description: 'number of results, between 1 and 99'
+
+          parameter name: :sort, in: :query, required: false, type: :string,
+                    description: "Comma separated sort field(s), prepend field(s) with '-' for descending sort"
+
+          parameter name: :filter, in: :query, required: false, type: :string,
+                    description: 'Filter on refill_status: [[refill_status][logical operator]=status]'
+
+          response 200 do
+            key :description, 'prescription index response'
+
+            schema do
+              key :'$ref', :PrescriptionsIndexResponse
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/prescriptions/active' do
+        operation :get do
+          key :description, 'Get a list of active prescriptions'
+          key :operationId, 'indexActivePrescriptions'
+          key :tags, %w(prescriptions)
+
+          parameter name: :page, in: :query, required: false, type: :integer,
+                    description: 'Page of results, greater than 0'
+
+          parameter name: :per_page, in: :query, required: false, type: :integer,
+                    description: 'number of results, between 1 and 99'
+
+          parameter name: :sort, in: :query, required: false, type: :string,
+                    description: "Comma separated sort field(s), prepend field(s) with '-' for descending sort"
```

Comment:
Seems like they are already being duplicated here. Is there an opportunity to consolidate these in some manner and DRY things up?